models/api.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require('../endpoints.json');
+
+exports.fetchEndpoints = () => {
```

Comment:
As this isn't related to the database per se, you can require in your JSON file in your controller and send it directly from there. Also, no need to use a promise, as the require will be completed before you send it anyway 👍